101-synapse-poc/azuredeploy.json

```diff
@@ -0,0 +1,621 @@
+                "07:00 AM (  7:00 )",
+                "08:00 AM (  8:00 )",
+                "09:00 AM (  9:00 )",
+                "10:00 AM ( 10:00 )",
+                "11:00 AM ( 11:00 )",
+                "12:00 PM ( 12:00 )",
+                "01:00 PM ( 13:00 )",
+                "02:00 PM ( 14:00 )",
+                "03:00 PM ( 15:00 )",
+                "04:00 PM ( 16:00 )",
+                "05:00 PM ( 17:00 )",
+                "06:00 PM ( 18:00 )",
+                "07:00 PM ( 19:00 )",
+                "08:00 PM ( 20:00 )",
+                "09:00 PM ( 21:00 )",
+                "10:00 PM ( 22:00 )",
+                "11:00 PM ( 23:00 )"
+            ],
+            "defaultValue": "09:00 PM ( 21:00 )"
+        },
+        "PauseTime": {
+            "type": "string",
+            "metadata": "Time of day when the data warehouse will be paused",
+            "allowedValues": [
+                "12:00 AM (  0:00 )",
+                "01:00 AM (  1:00 )",
+                "02:00 AM (  2:00 )",
+                "03:00 AM (  3:00 )",
+                "04:00 AM (  4:00 )",
+                "05:00 AM (  5:00 )",
+                "06:00 AM (  6:00 )",
+                "07:00 AM (  7:00 )",
+                "08:00 AM (  8:00 )",
+                "09:00 AM (  9:00 )",
+                "10:00 AM ( 10:00 )",
+                "11:00 AM ( 11:00 )",
+                "12:00 PM ( 12:00 )",
+                "01:00 PM ( 13:00 )",
+                "02:00 PM ( 14:00 )",
+                "03:00 PM ( 15:00 )",
+                "04:00 PM ( 16:00 )",
+                "05:00 PM ( 17:00 )",
+                "06:00 PM ( 18:00 )",
+                "07:00 PM ( 19:00 )",
+                "08:00 PM ( 20:00 )",
+                "09:00 PM ( 21:00 )",
+                "10:00 PM ( 22:00 )",
+                "11:00 PM ( 23:00 )"
+            ],
+            "defaultValue": "05:00 PM ( 17:00 )"
+        }
+    },
+    "variables": {
+        "defaultDataLakeStorageAccountUrl": "[concat('https://', variables('dlsName'), '.dfs.core.windows.net')]",
+        "synapseName": "[toLower(concat(parameters('companyTla'),parameters('deploymentType')))]",
+        "dlsName": "[toLower(concat('dls',parameters('companyTla'),parameters('deploymentType')))]",
+        "dlsFsName": "[toLower(concat(variables('dlsName'),'fs1'))]",
+        "sqlPoolName": "[toLower(concat(variables('workspaceName'),'p1'))]",
+        "workspaceName": "[toLower(concat(variables('synapseName'),'ws1'))]",
+        "sparkPoolName": "[toLower('synasp1')]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-10-01",
+            "name": "logicAppPauseDeployment",
+            "properties": {
+                "mode": "Incremental",
+                "templatelink": {
+                    "uri": "https://raw.githubusercontent.com/JamJarchitect/azure-quickstart-templates/master/101-synapse-poc/nestedtemplates/pausetemplate.json"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts